docs/clustering/downscaling.rst

```diff
@@ -0,0 +1,360 @@
+.. _scaling-down:
+
+===========
+Downscaling
+===========
+
+In this how-to guide you will:
+
+- Create a three node cluster which will run on a single host.
+- Add some data to it.
+- Scale it down to a single node cluster.
+
+.. _scaling-down-starting-vanilla-cluster:
+
+vanilla cluster
+===============
+
+Our cluster's name is ``vanilla``, it is composed of three CrateDB nodes, all running
+on the same host, and sharing the file system, operating system's scheduler, memory
+manager, and hardware.
+
+This setup provides a local cluster environment, when you only have one host, at the
+cost of increased latency in writes, because when operating as a cluster, the nodes
+must reach consensus on each write operation.
+
+To set it up I recommend any of our `deployment guides`_, using this template for each
+of the nodes:
+
+::
+
+   cluster.name: vanilla
+   node.name: <NODE_NAME>
+   network.host: _local_
+   node.max_local_storage_nodes: 3
+   stats.service.interval: 0
+
+   http.cors.enabled: true
+   http.cors.allow-origin: "*"
+
+   transport.tcp.port: <PORT>
```

Comment:
why is it needed to define this setting? afaik the default works fine.
Otherwise the settings for `discovery.seed_hosts` and `cluster.initial_master_nodes` would have to be adjusted as well.